pkg/utils/json_test.go

```diff
@@ -1,19 +1,40 @@
 package utils
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
-var _ = Describe("Testing utils helpers", func() {
-	Context("Test PrettyJSON", func() {
-		It("should return pretty JSON and no error", func() {
-			prettyJSON, err := PrettyJSON([]byte("{\"name\":\"baba yaga\"}"), "--prefix--")
-			Expect(err).ToNot(HaveOccurred())
-			Expect(prettyJSON).To(Equal([]byte(`{
+func TestPrettyJson(t *testing.T) {
+	assert := assert.New(t)
+
+	type prettyJSONtest struct {
+		input              []byte
+		prefix             string
+		expectedPrettyJSON []byte
+	}
+
+	prettyJSONtests := []prettyJSONtest{{
+		[]byte("{\"name\":\"baba yaga\"}"),
+		"--prefix--",
+		[]byte(`{
 --prefix--    "name": "baba yaga"
---prefix--}`)))
-		})
-	})
+--prefix--}`)},
+		{[]byte("should error"),
+			"",
+			nil},
+	}
+
+	for _, pjt := range prettyJSONtests {
+		prettyJSON, err := PrettyJSON(pjt.input, pjt.prefix)
 
-})
+		if err != nil {
```

Comment:
You don't need to assert.NotNil err here since we already know err is nil.

You could add a field to prettyJSONTest that is a bool for error and check if the error should have occurred. We need to ensure that the errors are happening for the correct cases